package Lesson.L19_Exceptions;

// Если метод а использует внутри себя метод b,который может выбросить checked исключение, то метод а должен:
// 1. Или заключить вызов метода b в try/catch блоки;
// 2. Или/и обявить, что он тоже может выбросить это checked исключение или его super класс
// 3.

public class Checked_Exception_2 {

}
